### DISCUSSION TITLE
Why is kit.vite.server.proxy not documented?

### DISCUSSION THREAD

**Discussion by kswope** [sveltejs/kit#2778] [Q&A]
I put this gem in my svelte.config.js

```
        proxy: {
          "/api": "http://127.0.0.1:4000/",
        },
```

And, like vue and react I can proxy my svelekit SPA to a backend API.

I only found this because of a 3 year svelte issue https://github.com/sveltejs/svelte/issues/3717

Is there a reason this isn't documented anywhere?  It seems like critical info for anybody developing a non-ssr SPA in sveltekit.

**Comment by benmccann** [comment]
That's a great hint!

It is documented in the Vite documentation (https://vitejs.dev/config/#server-proxy), which is linked to from the SvelteKit documentation (https://kit.svelte.dev/docs#configuration-vite). I don't know that we'd want to duplicate all the documentation between the two sources as it might be difficult to keep them in sync and up-to-date, but perhaps there are things we could do to make it more discoverable?
